$NetBSD: patch-src_Makevars,v 1.1 2026/06/10 13:17:47 mef Exp $

- The Problem is:
  RcppEigen.so: Undefined PLT symbol "backtrace_symbols" (symnum = 108)

- The symbol "backtrace_symbols" is defined in /usr/lib/libexecinfo
  at *BSD OS

- R package does not honor LIBS+= parameter, due to real build is done
  at (pkgsrc) install stage

- Neither bmake (.if eq ... .endif) nor gmake ( ifneq .. endif) does work
  in src/Makevars

- So the condition branch will be done at SUBST mechanism

--- src/Makevars.orig	2019-03-14 01:01:23.000000000 +0900
+++ src/Makevars	2026-06-10 21:56:52.450346394 +0900
@@ -1,3 +1,4 @@
 PKG_CPPFLAGS = -Iunix -I. -DRCPP_DEFAULT_INCLUDE_CALL=false
 PKG_CFLAGS = $(C_VISIBILITY)
 PKG_CXXFLAGS = $(CXX_VISIBILITY)
+@@PKG_LIBS@@
